website/docs/getting-started/installation.md

```diff
@@ -50,9 +50,17 @@ docker run --rm ghcr.io/oss-review-toolkit/ort:latest [command]
 
 ## From binaries
 
-Preliminary binary artifacts for ORT are currently available via
-[JitPack](https://jitpack.io/#oss-review-toolkit/ort). Please note that due to limitations with the JitPack build
-environment, the reporter is not able to create the Web App report.
+A binary distribution of ORT can be downloaded from the
+[latest GitHub release](https://github.com/oss-review-toolkit/ort/releases/latest). The `ort-[version].zip` file
```

Comment:
Since you're referring to "*A* binary distribution", not only the latest one, also "/latest" should be omitted from the URL to see all releases.

Reply:
I'm not sure I understand, the link is called `latest GitHub release`. I wanted to safe the user a click by directly pointing to it.

Reply:
Ah, right, sorry. I do not remember that I though when writing the comment. I probably misread it to point to a specific version.